pywinauto/unittests/test_common_controls.py

```diff
@@ -441,6 +464,43 @@ class ListViewTestCases64(ListViewTestCases32):
         path = os.path.join(mfc_samples_folder, u"RowList.exe")
 
 
+class ListViewWinFormTestCases32(unittest.TestCase):
+
+    """Unit tests for the ListViewWrapper class with WinForm applications"""
+
+    path = os.path.join(winform_folder_32, u"ListView_TestApp.exe")
+
+    def setUp(self):
+        """Set some data and ensure the application is in the state we want"""
+        Timings.Fast()
+
+        app = Application()
+        app.start(self.path)
+
+        self.dlg = app.ListViewEx
+        self.ctrl = app.ListViewEx.ListView.wrapper_object()
+
+    def tearDown(self):
+        """Close the application after tests"""
+        self.dlg.send_message(win32defines.WM_CLOSE)
+
+    def testCellClickInput(self):

```

Comment:
Test method names must follow PEP8 also. And please don't close pull request to make changes. Just commit to your fork, it will be reflected automatically.